contracts/MyMultiSig.cdc

```diff
@@ -39,13 +40,15 @@ pub contract MyMultiSigV2 {
     pub struct MessageSignaturePayload {
         pub let signingAddr: Address
         pub let message: String
+        pub let messageHex: String
         pub let keyIds: [Int]
         pub let signatures: [String]
         pub let signatureBlock: UInt64
 
-        init(signingAddr: Address, message: String, keyIds: [Int], signatures: [String], signatureBlock: UInt64) {
+        init(signingAddr: Address, message: String, messageHex: String, keyIds: [Int], signatures: [String], signatureBlock: UInt64) {
```

Comment:
instead of passing in Hex string, can we convert string to uft8, then encode to Hex?
`cadence•let utf8: [UInt8]`
`String.encodeHex(data)` (ref: https://developers.flow.com/cadence/language/values-and-types)